internal/proxy/search_util.go

```diff
@@ -21,6 +21,29 @@ import (
 	"github.com/milvus-io/milvus/pkg/util/typeutil"
 )
 
+func processPlaceholderGroup(t *searchTask) ([]byte, error) {
```

Comment:
Will it affect the query performance, considering the introduction of the unmarshal step?
How about deferring this check until the creation of the physical plan in cgo?

Reply:
Had the same concern here. The trade of will be no error return for empty segment